packages/core/core/src/BundleGraph.js

```diff
@@ -51,4 +51,20 @@ export default class BundleGraph extends Graph<BundleGraphNode> {
       }
     });
   }
+
+  isAssetReferenced(asset: Asset) {
```

Comment:
Currently unused. There is a hack later on to set a property when an asset is removed from a bundle since packagers currently don't have access to the BundleGraph. We need a way to know if an asset is used by another bundle...